acct-user/jenkins/jenkins-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="Jenkins program user"
+ACCT_USER_ID=473
+ACCT_USER_HOME=/var/lib/jenkins
```

Comment:
@mjo suggested using the default home to avoid fighting for the home directory by various packages.

Reply:
Sorry, I don't understand. Which default home should be used instead? Package used
```bash
JENKINS_DIR=/var/lib/jenkins

pkg_setup() {
	enewgroup jenkins
	enewuser jenkins -1 -1 ${JENKINS_DIR} jenkins
}
```
before and new acct-user/jenkins is mirroring this behavior. Not?

Reply:
It means that the acct-user/group package will only define the uid/gid, but the home directory will be specified in the dev-util/jenkins package itself as before.